src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_VALUE = 10.0;
+    private static final int MULTIPLAYER = 100;
+    private ColorSupplier colorSupplier;
+
+    public FigureSupplier() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
```

Comment:
it is better to bring this variable to the class level with the 'private' access modifier

Reply:
done